### DISCUSSION TITLE
Unable to connect to a mailbox such as Outlook or to a service that uses mail such as Kuma Uptime

### DISCUSSION THREAD

**Discussion by NashobaWellan** [Mailu/Mailu#3199] [User support]
I've set up mailu and can send and receive mail using the roundcube interface.

But as soon as I try to connect to Outlook using IMAP and SMTP, I get the error "We were unable to connect to the incoming server (IMAP)".

And for Kuma Uptime I received "connect ECONNREFUSED ip:465"

(Fetchmailis is on True)

Do you have any idea where this might be coming from?

**Comment by Diman0** [comment]
Maybe the ports are not opened. 
From the same computer as outlook/kuma runs on, try opening a telnet session on to the same hostname and port.
In the front logs (docker compose logs front) you can check if you see the incoming connections.

From your description I would assume that the connection does not reach the server that hosts Mailu.
